apps/gateway/src/routes/v2/index.ts

```diff
@@ -0,0 +1,9 @@
+import { createRouter } from '$/openApi/createApp'
+import { chatHandler, chatRoute } from './conversations/chat'
+import { runHandler, runRoute } from './documents/run'
+
+const router = createRouter()
+  .openapi(runRoute, runHandler)
+  .openapi(chatRoute, chatHandler)
+
+export default router
```

Comment:
I don't know why we don't do the same in v3. Just one index file to be imported. Can you check if can be done? again, this is the way i did in v2 move to openapu, so my bad